javascript-data-model/order-history.js

```diff
@@ -0,0 +1,49 @@
+const orderHistory = [
+  {
+    title: 'JavaScript for Impatient Programmers',
+    author: 'Dr. Axel Rauschmayer',
+    total: 34,
+    'order #': 11439416898772232,
+    'order placed': 'August 4, 2020',
+    delivered: 'August 8, 2020',
+    'return window closed': 'September 7, 2020',
+  },
+  {
+    title: 'The Timeless Way of Building',
+    author: 'Christopher Alexander',
+    total: 44.53,
+    'order #': 1139984268120257,
+    'order placed': 'July 19, 2020',
+    delivered: 'July 20, 2020',
+    'return window closed': 'August 19, 2020',
+  },
+  {
+    title: 'GameCube Controller Adapter',
+    author: 'Knockoff',
+    total: 17.22,
+    'order #': 1142875557905940,
+    'order placed': 'July 4, 2020',
+    delivered: 'July 7, 2020',
+    'return window closed': 'August 5, 2020',
+  },
+  {
+    title: 'GameCube Controller - Super Smash Bros. Edition',
+    author: 'Nintendo',
+    total: 94.95,
+    'order #': 11328831772648248,
+    'order placed': 'July 3, 2020',
+    delivered: 'July 5, 2020',
+    'return window closed': 'August 4, 2020',
+  },
+  {
+    title: 'The Art of SQL',
+    author: 'Stephane Faroult',
+    total: 33.99,
+    'order #': 11328831772648248,
+    'order placed': 'July 3, 2020',
+    delivered: 'July 5, 2020',
+    'return window closed': 'August 4, 2020',
+  },
```

Comment:
Since these are apart of the same order, they shouldn't be 2 different objects. Each object inside this array should be each order, not each product.

A better approach would probably be have 1 object for the whole order, then have a `products` property which is an array of objects for the different products that were bought. That way you can store a bunch of information that's unique for each item, but still apart of the same order.